### PR TITLE
[AKS] remove patterns for adminUsername and adminPassword in WindowsProfile

### PR DESCRIPTION
**Pull request by AbelHu**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
AKS

File: src/aks-preview/azext_aks_preview/_help.py [hunk 1]
```diff
@@ -67,9 +67,25 @@
         - name: --windows-admin-username
           type: string
           short-summary: User account to create on windows node VMs.
+          long-summary: |-
+            Rules for windows-admin-username:
+                - restriction: Cannot end in "."
+                - Disallowed values: "administrator", "admin", "user", "user1", "test", "user2", "test1", "user3", "admin1", "1", "123", "a", "actuser", "adm", "admin2", "aspnet", "backup", "console", "david", "guest", "john", "owner", "root", "server", "sql", "support", "support_388945a0", "sys", "test2", "test3", "user4", "user5".
```
Comment (zqingqing1) [inline src/aks-preview/azext_aks_preview/_help.py:73]: do we have public doc about this disallowed value? if yes, can we link the doc here?

File: src/aks-preview/azext_aks_preview/_help.py [hunk 2]
```diff
@@ -67,9 +67,25 @@
...
+          long-summary: |-
+            Rules for windows-admin-password:
+                - Minimum-length: 8 characters
+                - Max-length: 123 characters
+                - Complexity requirements: 3 out of 4 conditions below need to be fulfilled
+                  * Has lower characters
+                  * Has upper characters
+                  * Has a digit
+                  * Has a special character (Regex match [\\W_])
+                - Disallowed values:  "abc@123", "P@$$w0rd", "P@ssw0rd", "P@ssword123", "Pa$$word", "pass@word1", "Password!", "Password1", "Password22", "iloveyou!"
```
Comment (zqingqing1) [inline src/aks-preview/azext_aks_preview/_help.py:88]: same here, link a doc if we have.

Comment (zqingqing1) [reply src/aks-preview/azext_aks_preview/_help.py:88]: seems like our backend service will do the validation?

Comment (AbelHu) [reply src/aks-preview/azext_aks_preview/_help.py:89]: This is to unblock customers to use `-` in the username. Currently AKS relies on CRP to do the validation. AKS will add the validation soon.

**Comment by AbelHu** [comment]
@yonzhan could you help to take a look at the test failure in "Azure.azure-cli-extensions (Verify Extensions Index) "?

**Review by zqingqing1 — APPROVED** [review]
lgtm

**Review by arrownj — APPROVED** [review]
LGTM

**Pull request merged**